### ISSUE TITLE
Attempting to skip NodePlugin results in: "You tried to skip plugins that would never have been loaded anyway"

### ISSUE THREAD

**Issue by charlex** [graphile/graphile-engine#607]
```
A serious error occurred when building the initial schema (attempt 2). We'll try again in 400ms. Error details:
Error: You tried to skip plugins that would never have been loaded anyway. Perhaps you've made a mistake in your skipPlugins list, or have sourced the plugin from a duplicate plugin module - check for duplicate modules in your 'node_modun
    at Object.exports.getPostGraphileBuilder (/xxx/node_modules/postgraphile-core/src/index.ts:372:11)
    at Object.exports.watchPostGraphileSchema (/xxx/node_modules/postgraphile-core/src/index.ts:469:25)
    at createGqlSchema (/xxx/node_modules/postgraphile/src/postgraphile/postgraphile.ts:94:17)
    at Object.enhanceHttpServerWithSubscriptions (/xxx/node_modules/postgraphile/src/postgraphile/http/subscriptions.ts:71:18)
A serious error occurred when building the initial schema (attempt 3). We'll try again in 900ms. Error details:
```

```
import { NodePlugin } from 'graphile-build';

import {
  postgraphile,
  makePluginHook,
  PostGraphileOptions,
  Middleware,
  enhanceHttpServerWithSubscriptions,
  Plugin,
} from 'postgraphile';

export function getPostGraphileOptions({
  websocketMiddlewares,
  rootPgPool,
}: IPostGraphileOptionsOptions) {
  const options: PostGraphileOptions<Request, Response> = {
    /*
     * Plugins we don't want in our schema
     */
    skipPlugins: [
      // Disable the 'Node' interface
      NodePlugin as Plugin,
    ],
  }
}
```

**Comment by benjie** [comment]
This probably means you have more than one version of graphile-build in node_modules. Use yarn / yarn resolutions / yarn-deduplicate to resolve.

[semi-automated message] Thanks for your question; hopefully we're well on the way to helping you solve your issue. This doesn't currently seem to be a bug in the library so I'm going to close the issue, but please feel free to keep requesting help below and if it does turn out to be a bug we can definitely re-open it 👍

You can also ask for help in the #help-and-support channel in [our Discord chat](http://discord.gg/graphile).

**Comment by charlex** [comment]
Thank you @benjie, this worked.

```
  "resolutions": {
    "graphile-build": "4.5.x"
  },
```

**Comment by Venryx** [comment]
In my case, the problem was from these two copies of `graphile-utils`:
* `node_modules/graphile-utils`
* `node_modules/postgraphile/node_modules/graphile-utils`

I tried removing my direct dependency on `graphile-utils`, but even after that, yarn was not hoisting the `postgraphile/node_modules/graphile-utils` to the root level.

Anyway, to resolve the issue, I removed the direct dependency on `graphile-utils`, and just imported from `node_modules/postgraphile/node_modules/graphile-utils` instead. (I know this is a bit fragile, since it depends on the package manager's installation algorithm, but I prefer it over the yarn resolutions approach in this case)

**Comment by Venryx** [comment]
Oh, I found out the reason that there were two copies of `graphile-utils`. It was similar to this: https://twitter.com/dan_abramov/status/951931842273398784

Basically, postgraphile was using a different version of graphql than my root project, so yarn was not able to hoist postgraphile's `graphile-utils` subdependency to the root level, even if the version matched the version of itself in the root `node_modules`.

Once I added a `resolutions` version-lock for `graphql` as well, the duplicate `graphile-utils` (at `node_modules/postgraphile/node_modules/graphile-utils`) went away.

This is a better solution than the approach in my previous post, so I'm using it now.
